### PR TITLE
Add some more cluster startup printing and initdb progress info #5

### PR DESCRIPTION
**Pull request by bmatican**
Sample create and output:
```
./bin/yb-ctl --data_dir /tmp/yugabyte-local-cluster  --rf 3 create
Starting cluster
Waiting for cluster to be ready.
Initializing cluster
..................................................................................
----------------------------------------------------------------------------------------------------
| Number of servers: 3 | Replication factor: 3                                                     |
----------------------------------------------------------------------------------------------------
| JDBC                : postgresql://postgres@127.0.0.1:5433                                       |
| YSQL                : ./bin/psql -U postgres -h 127.0.0.1 -p 5433                                |
| YCQL                : ./bin/cqlsh 127.0.0.1 9042                                                 |
| YEDIS               : ./bin/redis-cli -h 127.0.0.1 -p 6379                                       |
| Web UI              : http://127.0.0.1:7000/                                                     |
| Cluster Data        : /tmp/yugabyte-local-cluster                                                |
----------------------------------------------------------------------------------------------------
```

### CODE REVIEW

File: bin/yb-ctl [hunk 1]
```diff
@@ -1134,6 +1148,7 @@ class ClusterControl:
         return True
 
     def wait_for_cluster_or_raise(self):
+        print("Waiting for cluster to be ready.")
```
Comment (ndeodhar) [inline bin/yb-ctl:1151]: nit: Remove the period from the end to keep it consistent with the other two statements. "Starting cluster" and "Initializing cluster" don't end with a period, so this looks inconsistent.

File: bin/yb-ctl [hunk 2]
```diff
@@ -1401,12 +1412,13 @@ class ClusterControl:
...
                 with open(initdb_log_path, 'w') as initdb_stdout_file:
-                    subprocess.check_call([
+                    proc = subprocess.Popen([
                         initdb_binary_path,
                         '-D', tmp_pg_data_dir,
                         '-U', 'postgres'
                     ], stdout=initdb_stdout_file, stderr=subprocess.STDOUT)
-                succeeded = True
+                    wait_for_proc_report_progress(proc)
+                    succeeded = proc.returncode == 0
```
Comment (ndeodhar) [inline bin/yb-ctl:1421]: Indentation for these two lines looks off. Shouldn't it be aligned with "with" on line 1414.

**Review by ndeodhar — COMMENTED** [review]
Minor comments. Otherwise, LGTM.

File: bin/yb-ctl (hunk 1 above)
Comment (bmatican) [reply bin/yb-ctl:1151]: oops, fair point

File: bin/yb-ctl (hunk 2 above)
Comment (bmatican) [reply bin/yb-ctl:1421]: actually I think this is probably ok, since I do not initialize proc outside of the `with` block.

**Comment by mbautin** [comment]
@bmatican Do we need this pull request or has this been incorporated into some other PR that got merged?

**Comment by bmatican** [comment]
@mbautin , I believe @ndeodhar ported this over to https://github.com/YugaByte/yugabyte-installation/pull/7 when I was out and merged it in, so I can close this.

**Pull request closed without merging**